### PR TITLE
Fix for issue where sortable is trying to load when not needed

### PR DESCRIPTION
**Pull request by Prestaworks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fix for issue where sortable is trying to load when not needed.
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/PrestaShop#31980.
| How to test?  | Follow instructions in ticket.

### CODE REVIEW

**Review by leemyongpakvn — COMMENTED** [review]
I think combining with existing condition is better
```
        if (Tools::getValue('controller') !== 'AdminModules' 
            || Tools::getValue('configure') !== $this->name 
            || Tools::getValue('id_slide')) {
                return;
        }
```

**Comment by kpodemski** [comment]
Hi @Prestaworks 

I agree with @leemyongpakvn suggestion :-)

File: ps_imageslider.php [hunk 1]
```diff
@@ -618,9 +618,11 @@ public function hookActionShopDataDuplication($params)
...
     public function headerHTML()
     {
-        if (Tools::getValue('controller') != 'AdminModules' && Tools::getValue('configure') != $this->name) {
-            return;
-        }
+        if ('AdminModules' !== Tools::getValue('controller') ||
+            Tools::getValue('configure') !== $this->name ||
+            Tools::getIsset('id_slide')) {
+                return;
+       }
```
Comment (leemyongpakvn) [inline ps_imageslider.php:625]: It looks like lack of 1 space at line 625

**Review by leemyongpakvn — APPROVED** [review]
We have `Tools::` on both sides now ;)

**Review by 0x346e3730 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @Prestaworks 

I have checked your PR. 

**Check on PS8.0.3** : An error is displayed which is related to permission. It's Ok when giving permission to the shop 

- Check on BO : OK
- Check on FO : OK

https://user-images.githubusercontent.com/92912932/234547041-68bea1d1-18e7-424b-ad53-59fadd3a4b67.mp4


**Check on PS1.7.8.8** : The JS error is still displayed when editing the slide 

image

https://user-images.githubusercontent.com/92912932/234546837-3f15da86-8044-45aa-aa2e-a02468eeedb8.mp4

Could you please check it ?

Thank You!

**Comment by Prestaworks** [comment]
Hi @HanaRebaiQA ,
if the PR is applied properly, no javascripts are added from the module on the "edit slide" page. So that JS error should be from something else.

**Comment by leemyongpakvn** [comment]
I guess **HanaRebaiQA** need fresh installs of PS 8.0.3, PS 1.7.8.8, and ps_imageslider dev to make sure this PR is not affected by other PR. I rarely test other contributors' PR, then I see this PR works as supposed on my nearly fresh PS 8.1 :)

**Comment by nicosomb** [comment]
Thank you @Prestaworks !

**Comment by samilmarekrygula** [comment]
I have added this extra condition also, because it was problem when you wanted to add new slider Tools::getIsset('addSlide')

**Pull request merged**